### PR TITLE
Fix model description getting lost

### PR DESCRIPTION
**Pull request by bkniffler**

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by bajtos** [comment]
@slnode ok to test

File: lib/model-helper.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ var modelHelper = module.exports = {
 
     out[name] = {
       id: name,
-      description: typeConverter.convertText(def.description),
+      description: typeConverter.convertText(def.settings.description),
```
Comment (bajtos) [inline lib/model-helper.js]: FWIW, this fails when `def.settings` is not defined, see existing unit-tests for an example.

Comment (bajtos) [inline lib/model-helper.js]: Fixed by ccfd9c42

**Pull request merged**